build/yaml/pipelines/teams-js.yml

```diff
@@ -0,0 +1,20 @@
+name: $(Build.BuildId)
```

Comment:
My initial thought with the addition of the Language variable and this file is that we then end up with a lot of largely duplicate files. I don't think I would change the approach we're taking here for right now, but I would probably add this to the list of things to review post-R13 and see how we feel about it.

Reply:
Yeah, we should probably reconcile the structure of multi-language packages, then make the templates smart enough to look for `${LANGUAGE}/*` and run the appropriate commands.